presto-orc/src/test/java/com/facebook/presto/orc/metadata/statistics/TestBinaryStatisticsBuilder.java

```diff
@@ -122,4 +112,11 @@ private void assertBinaryStatistics(ColumnStatistics columnStatistics, int expec
             assertEquals(columnStatistics.getNumberOfValues(), 0);
         }
     }
+
+    public static void addValue(BinaryStatisticsBuilder binaryStatisticsBuilder, Slice slice)
+    {
+        int[] offsets = new int[] {0, slice.length()};
+        VariableWidthBlock block = new VariableWidthBlock(1, slice, offsets, Optional.empty());
```

Comment:
I'd prefer to use the builder to avoid any unnecessary changes in case if VariableWidthBlock constructor is changed.